exe/kubernetes-deploy

```diff
@@ -15,18 +15,23 @@ require 'kubernetes-deploy'
 require 'optparse'
 
 skip_wait = false
+template_dir = nil
 ARGV.options do |opts|
   opts.on("-w", "--skip-wait") { skip_wait = true }
+  opts.on("-tDIR", "--template-dir=DIR") { |v| template_dir = v }
   opts.parse!
 end
 
+if !template_dir || template_dir.empty?
+  raise ArgumentError, "--template-dir flag is required"
+end
```

Comment:
I see your point, though I'm not convinced that the default belongs in the other repo more than it does here. You could also eliminate the arg to `Runner.new` by setting the default right here, no? I don't feel super strongly about this though.